msteams-platform/sbs-gs-blazorupdate.yml

```diff
@@ -332,6 +332,7 @@ items:
 
        1. **Subscription name**: Visual Studio Enterprise Subscription. </br>
        1. **Resource group**: Select **New** to add the resource group generated for your app.</br>
+
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
Removed the space as suggested.